wordpress-mobile/WordPress-iOS
# Migrates 4 services to WordPressKit

Fixes #7305 #7306 #7329 #7330 

This PR migrates:

* The `BlogServiceRemote` remote service class 
* The `BlogServiceRemoteREST ` remote service class 
* The `BlogServiceRemoteXMLRPC ` remote service class 
* The `SiteManagementServiceRemote ` remote service class
* The `SiteServiceRemoteWordPressComREST ` remote service class 
* Basic (non-managed) models related to 👆 
* Related unit tests + mock data

to the WordPressKit framework. In addition, the unit tests were rewritten for `SiteManagementServiceRemote` (and reside in WPKit). Ref #7301

**To test:**

1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green
5. Run the app and log in with a .com account. Verify things are working properly.
6. Run the app and log in with a .org account. Verify things are working properly.

Needs review: @astralbodies
